canvas_manage_course/settings/isites_migration.py

```diff
@@ -1,17 +1,32 @@
+"""
+These settings are used for:
+- isites_migration.management.commands.export_slide_tool (just exporting slide
+    tool data; uses e.g.
+- jobs put on the RQ isites_export queue (which run
+    isites_migration.jobs.migrate_files)
+"""
 from .aws import *
 
 INSTALLED_APPS = (
-    'icommons_common',
     'async',
     'django_rq',
+    'icommons_common',
     'isites_migration',
+    'lti_school_permissions',
```

Comment:
Looks like this is part of moving up to Django 1.9, which was part of the most recent release. https://github.com/Harvard-University-iCommons/canvas_manage_course/commit/12dab87d3cd1f9bc5acb6d5c47c2322dc69ab7ad

The issue appears to be that the Django startup checks now fail if models are referenced in apps that aren't part of `INSTALLED_APPS` and don't have `app_label`s in their model `Meta:` data.
https://docs.djangoproject.com/en/1.10/ref/models/options/#app-label

I verified that adding `app_label`s fixes this issue and will submit PRs to clean this up.

Here's an example stack trace of the issue once you move to Django 1.9,  don't have these in `INSTALLED_APPS`, and they don't have `app_label`s.

```
Traceback (most recent call last):
File "/home/vagrant/tlt/canvas_manage_course/manage.py", line 13, in <module>
execute_from_command_line(sys.argv)
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/core/management/init.py", line 353, in execute_from_command_line
utility.execute()
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/core/management/init.py", line 345, in execute
self.fetch_command(subcommand).run_from_argv(self.argv)
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/core/management/base.py", line 348, in run_from_argv
self.execute(*args, **cmd_options)
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/core/management/base.py", line 398, in execute
self.check()
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/core/management/base.py", line 426, in check
include_deployment_checks=include_deployment_checks,
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/core/checks/registry.py", line 75, in run_checks
new_errors = check(app_configs=app_configs)
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/core/checks/urls.py", line 13, in check_url_config
return check_resolver(resolver)
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/core/checks/urls.py", line 23, in check_resolver
for pattern in resolver.url_patterns: File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/utils/functional.py", line 33, in get
res = instance.dict[self.name] = self.func(instance)
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 417, in url_patterns
patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/utils/functional.py", line 33, in get
res = instance.dict[self.name] = self.func(instance)
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 410, in urlconf_module
return import_module(self.urlconf_name)
File "/usr/lib/python2.7/importlib/init.py", line 37, in import_module
import(name)
File "/home/vagrant/tlt/canvas_manage_course/canvas_manage_course/urls.py", line 3, in <module>
from canvas_manage_course import views
File "/home/vagrant/tlt/canvas_manage_course/canvas_manage_course/views.py", line 15, in <module>
from lti_school_permissions.decorators import (
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/lti_school_permissions/decorators.py", line 8, in <module>
from lti_school_permissions.verification import is_allowed
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/lti_school_permissions/verification.py", line 12, in <module>
from lti_school_permissions.models import SchoolPermission
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/lti_school_permissions/models.py", line 7, in <module>
class SchoolPermission(models.Model):
File "/home/vagrant/.virtualenvs/canvas_manage_course/local/lib/python2.7/site-packages/django/db/models/base.py", line 102, in new
"INSTALLED_APPS." % (module, name)
RuntimeError: Model class lti_school_permissions.models.SchoolPermission doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```